src/apps/an-delay/screens/DelayDetail.js

```diff
@@ -88,6 +91,10 @@ const DelayDetail = React.memo(({ delay, path, targetApp, loading }) => {
     setModalVisible(false)
   }, [])
 
+  const isFromSignalingProposal =
+    path.length > 0 &&
+    path[0].description.indexOf(VOTING_DESCRIBED_STEP_PREFIX) !== -1
+
```

Comment:
Maybe we can reuse the `isEmptyCallScript(...)` [function](https://github.com/BlossomLabs/aragon-dao/blob/3bdf9fdb28298b832110f6ec4dc81282e23d6f31/src/utils/evmscript.js#L9) instead of reimplementing this logic.

We can get the script from the `delay` props.